tests/compilation/test_compiling_contract_sources.py

```diff
@@ -32,12 +32,24 @@ def test_compiling_project_contracts(project):
 @load_contract_fixture('ImportTestA.sol')
 @load_contract_fixture('ImportTestB.sol')
 @load_contract_fixture('ImportTestC.sol')
+@load_contract_fixture('ImportTestD.sol')
+@load_contract_fixture('ImportTestE.sol')
+@load_contract_fixture('ImportTestF.sol')
```

Comment:
See #314 for a strategy for testing import remappings.  The three tests that I see as valuable are:

1. A test exactly like what is present in #314 where a contract within `./contracts` imports another contract that can be found under `./contracts` but using a remapped import path.
2. A test where a contract under `./contracts` imports a contract that is within the `project_dir` but **not** under `./contracts` using a remapped import path.
3. A test where a contract under `./contracts` imports a contract that is *outside* of the `project_dir` using a remapped import path.

All three of these are well suited to only being tested using relative paths, but if there is a *nice* way to test both relative and absolute paths that would be fine, just not necessary at this time.